lib/src/log_factory.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:collection';
```

Comment:
The import is automatic. Is needed for:

https://github.com/gmpassos/logger/blob/master/lib/src/log_factory.dart#L56

I only can ensure the order of the outputs using a `LinkedHashSet`, since collection literals does not guarantee that the hash implementation is ordered. Common hash implementations have random key ordering (`hashCode` dependent).

Note that in the browser the hash literal has ordered keys, but there's nothing that can guarantee that in other platforms, like flutter that is compiled to iOS and Android. Actually I see that Dart behavior as a hot spot for bugs, since many tests are executed in a single platform (at least unit tests should force a hash implementation with random keys).